content/sensu-core/2.0/getting-started/glossary.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Glossary of Terms"
+linkTitle: "Glossary"
+description: "Introduction to Sensu terms and concepts"
+weight: 10
+version: "2.0"
+product: "Sensu Core"
+menu:
+  sensu-core-2.0:
+    parent: getting-started
+---
+
+#### Agent
+A lightweight client that runs on the infrastructure components you want to monitor. Agents self-register with the backend, send keepalive messages, and execute monitoring checks. Each agent belongs to one or more subscriptions that determine which checks the agent runs. An agent can run checks on the entity it’s installed on or by connecting to a remote proxy entity. [Read more.][1]
+
+#### Backend
+A flexible, scalable monitoring event pipeline. The backend processes event data using filters, mutators, and handlers. It maintains configuration files, stores recent event data, and schedules monitoring checks. You can interact with the backend using the API, command line, and dashboard interfaces. [Read more.][2]
+
+#### Check
+A script run by the agent to determine the state of a system component or collect metrics. Checks specify an interval for execution, one or more subscriptions, and one or more handlers that will process the resulting event data. Once run, a check produces a check result that includes the output of the command and an exit status code. [Read more.][3]
+
+#### Check asset
+A script that a check can specify as a dependency. Assets must be a tar archive (optionally gzipped) with scripts or executables within a bin folder. At runtime, the agent installs required assets using the specified URL. Assets let you manage runtime dependencies without using configuration management tools. [Read more.][4]
+
+#### Check hook
+A command executed by the agent in response to a check result, before creating a monitoring event. Hooks create context-rich events by gathering related information based on the exit status code. [Read more.][5]
+
+#### Check plugin
+Checks require a script that the agent runs on the corresponding entity when executing the check. You can write your own check scripts in Go, Ruby, Python, and more, or use one of over 100 check plugins shared by the Sensu community. [Read more.][6]
```

Comment:
Sorta goes back to my comment above where a Check Plugin is the executable, but a Check would be the whole definition including the Check Plugin.